docker/.env

```
@@ -266,7 +271,12 @@ SMQ_VERIFICATION_URL_PREFIX=http://localhost/verify-email
 SMQ_VERIFICATION_EMAIL_TEMPLATE=verification-email.tmpl
 
 #### Users Client Config
-SMQ_USERS_URL=users:9002
+SMQ_USERS_URL=http://users:9002
+SMQ_USERS_GRPC_URL=users:7002
+SMQ_USERS_GRPC_TIMEOUT=300s
+SMQ_USERS_GRPC_CLIENT_CERT=${GRPC_MTLS:+./ssl/certs/domains-grpc-client.crt}
```

dotenv-linter (4.0.0)

[warning] [SubstitutionKey] The SMQ_USERS_GRPC_CLIENT_CERT key is not assigned properly

(SubstitutionKey)